### DISCUSSION TITLE
22.07

### DISCUSSION THREAD

**Discussion by falkTX** [DISTRHO/Cardinal#313] [Announcements]
See official release announcement at https://kx.studio/News/?action=view&url=cardinal-2207-released

- Adjust view menu similar to VCV Rack 2.1.2
- Allow building using OpenGL ES2 (with `USE_GLES=true` build flag, disables glBars module)
- Do not install JACK standalone on Windows by default
- Fix example patches not working as templates
- Fix file dialogs not working in macos-intel builds
- Fix missing opus as supported extension in audio file module
- Fix missing time information in AU version (regression in 22.06)
- Fix VST3 keyboard input through host keycodes
- Fix VST3 not loading under Cubase
- Fix VST3 UI not appearing under Studio One
- Give keyboard focus to UI on each mouse click
- Make X11 clipboard handling more robust
- Update all modules that use file browser dialogs
- Update internal Cardinal async file dialog API to add default filename
- Update to Rack 2.1.2
 
Web version:
- Initial, experimental support for web-assembly
- Available at https://cardinal.kx.studio/ for demo purposes
- Built as-is from current source code using emscripten, with a few LV2 and JSFX plugins added in
- Part of official releases as Cardinal-wasm-22.07.zip, easily rehosted (pure static files)
- Related discussion at https://github.com/DISTRHO/Cardinal/issues/287

<hr /><em>This discussion was created from the release <a href='https://github.com/DISTRHO/Cardinal/releases/tag/22.07'>22.07</a>.</em>

**Comment by FalaghSepehr** [comment]
Hi. I have a problem opening the cardinal modular plugin. Windows 7 x64. Ableton Live 10 & FL Studio 20.

I've tried 22.07 (Cardinal-win64-22.07.zip) and the previous version (Cardinal-win64-22.06-installer.exe) 

Ableton can not find the plugin (both versions). I've tried it with FL Studio too and the status of the plugin is "error" and it can't open them(both versions).

I tried to open the standalone version too (22.07) and there's an error: "Failed to create JACK client, reason was: Communication error with the JACK server". 

Please help! this is such a great modular plugin. I've watched a lot of videos about modular plugins and Cardinal is the best one and has the best interface.

**Reply by FalaghSepehr** [reply]
Here are the screenshots: 
![JACK Server Error](https://user-images.githubusercontent.com/112719861/188206711-405adb26-19cb-4e04-995a-2b326bd2a2b9.PNG)
![FL Studio Error](https://user-images.githubusercontent.com/112719861/188206835-d127816d-5c92-4acc-8782-67f093b63cda.PNG)

**Reply by falkTX** [reply]
try with the latest nightly builds, like https://github.com/DISTRHO/Cardinal/actions/runs/2977830754

there is now a native audio standalone, that at least should work..

**Reply by FalaghSepehr** [reply]
Does this also fix the plugin version of cardinal??

**Reply by falkTX** [reply]
no idea, this is the first time I see such an issue. and the error is quite generic, it is impossible to know why it fails.

**Reply by FalaghSepehr** [reply]
Ok, thank you so much for the feedback!

**Reply by FalaghSepehr** [reply]
> 

Also, is cardinal resizable?

**Reply by falkTX** [reply]
yes. on plugin formats that do not allow the host to resize the window, like vst2, there is a resize handle on the bottom-right to do this from the plugin UI side.

and btw, if nothing else works for you maybe https://cardinal.kx.studio/ will :)
